get.c

```diff
@@ -0,0 +1,19 @@
+#include "main.h"
```

Comment:
Filename `get.c`: Is it a get or a translation from one type (color pair) to another (number)?